index.js

```diff
@@ -51,38 +52,52 @@ function DB (opts) {
   self.refs = join({
     log: self.log,
     db: sub(self.db, 'r'),
-    map: function (row) {
+    map: function (row, cb) {
       if (!row.value) return
       var k = row.value.k, v = row.value.v || {}
       var ops = []
-      var refs = v.refs || row.value.refs || []
-      refs.forEach(function (ref) {
-        row.links.forEach(function (link) {
-          ops.push({ type: 'del', key: ref, rowKey: link })
-        })
-        if (k) ops.push({ type: 'put', key: ref, value: k })
+      var next = after(function () {
+        cb(null, ops)
       })
+
+      // Update refs
+      var refs = v.refs || row.value.refs || []
       var members = v.members || row.value.members || []
-      members.forEach(function (member) {
-        if (typeof member === 'string') member = { ref: member }
```

Comment:
This was for backwards compatibility, but at least none of our users created relations when this structure was used (way back in early version) so I think we can drop this, but this is "strictly" a breaking change in database structure.